### ISSUE TITLE
spreadsheets

### ISSUE THREAD

**Issue by almost-jay** [almost-jay/hierarchical_notebook_app#59]
The persistent note for each entry should be able to have a spreadsheet added to it by pressing the Add Spreadsheet button. Spreadsheets are saved as .csv files and have basic formulas and keyboard navigation functionality.

(also design better integration of the spreadsheets with the main entry stream)
